js_modules/dagster-ui/packages/ui-core/src/assets/AssetPageHeader.oss.tsx

```diff
@@ -72,10 +72,7 @@ export const AssetPageHeader = ({
   return (
     <PageHeader
       title={
-        <Box
-          flex={{alignItems: 'center', gap: 4}}
-          style={{maxWidth: '600px', overflow: 'hidden', marginBottom: 4}}
-        >
+        <Box flex={{alignItems: 'center', gap: 4}} style={{maxWidth: '600px', marginBottom: 4}}>
```

Comment:
Interesting - verified this wasn't from Dish's latest PR about the truncating breadcrumbs, so it seems fine to remove to me 👍